src/org/gk/render/DefaultComplexRenderer.java

```diff
@@ -23,6 +25,17 @@ public DefaultComplexRenderer() {
     }
     
     protected void renderShapes(Graphics g) {
+        // If node is a drug.
+        if (InstanceUtilities.isDrug(node.getReactomeId())) {
```

Comment:
I think it will be more reliable to add a flag in the Node class instead of checking this on the fly. See my comment for isDrug().

Reply:
Fixed in [d035031](https://github.com/reactome/CuratorTool/pull/26/commits/7ed99c3714987c8902ec7d8dfdeae55c3d035031).